ui/app/src/Utils/Api.elm

```diff
@@ -66,7 +66,7 @@ iso8601Time =
 
 baseUrl : String
 baseUrl =
-    "/api/v1"
+    "http://localhost:9093/api/v1"
```

Comment:
Would be nice to configure this from [the app flags](http://package.elm-lang.org/packages/elm-lang/html/1.1.0/Html-App#programWithFlags), so a `go` backend would pass it down to the html template.